gd1_helpers/membership/run_full_gd1_model.py

```diff
@@ -0,0 +1,258 @@
+import astropy.table as at
```

Comment:
Move this script into a new directory in the top level of the repo: `scripts/run_full_gd1_model.py`